Automattic/jetpack
# Social: Fix bug with video restrictions

There is a bug in the media picker, where you get an error when during publish the connections get disabled, and the video reduce would run on an empty array. Adding `[]` as an initial value fixes it.

### Other information:


## Jetpack product discussion



p1677764418312049-slack-C02JJ910CNL

## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






Test publishing with Publicize, you should not get an error.

Run the tests with:
`npm run test -- src/hooks/use-media-restrictions/test/index.test.js` in `projects/js-packages/publicize-components`

It should not fails. Remove the initial value, then it should fail
